src/crystal/system/win32/mime.cr

```diff
@@ -0,0 +1,6 @@
+module Crystal::System::MIME
+  # Load MIME types from operating system source.
+  def self.load
+    # stub
```

Comment:
Replace this with a comment saying that windows supplies no mime types.

Reply:
Well it does, You'll just need to be able to access the registry to get them. But I will clarify that and add a TODO.